src/pages/console/storage/file-storage.mdx

```diff
@@ -15,15 +15,17 @@ When you create new file storage for your app, you can edit authorization rules.
 
 1. Sign in to the AWS Management console and open AWS Amplify.
```

Comment:
feel this info is redundant and can be removed in favor of just saying launch studio and linking the start page